addons/web/static/src/scss/ui.scss

```diff
@@ -13,3 +13,8 @@
 [aria-hidden="true"], [aria-hidden="1"] {
     display: none!important;
 }
+
+.btn-unstylized {
```

Comment:
Basically, this is equal to the `btn bg-transparent` combo with BS4... but well, I suppose it could be ok.